lib/FusionInventory/Agent/Tools/Generic.pm

```diff
@@ -91,6 +92,54 @@ sub getDmidecodeInfos {
     return $info;
 }
 
+sub getIpmiFru {
```

Comment:
You should copy back this tool function in FusionInventory::Agent::Task::Inventory::Generic::Ipmi::Fru module as _getIpmiFru()